ui/src/app/pages/profile/data-use-agreement-content.tsx

```diff
@@ -0,0 +1,200 @@
+          </li>
+        </IndentedUnorderedList>
+      </li>
+    </IndentedUnorderedList>
+    <SecondHeader>As “Authorized Demonstration User” of the <AoUTitle/> data, I
+      will:</SecondHeader>
+    <IndentedUnorderedList>
+      <li><strong>NOT</strong> share my login information with anyone.
+        <IndentedUnorderedList>
+          <li>I will not share my login information with another authorized demonstration or other user
+            of the <AoUTitle/>.
+          </li>
+          <li>I will not create any group or shared accounts.</li>
+        </IndentedUnorderedList>
+      </li>
+      <li>
+        <strong>NOT</strong> use <AoUTitle/> data or any external data, files, or software that I upload
+        into the Research Workbench for research that is discriminatory or stigmatizing of individuals, families,
+        groups, or communities. Please review the All of Us policy on stigmatizing research here.
+        <IndentedUnorderedList>
+          <li>I will contact the <AoUTitle/> Resource Access Board (RAB) for
+            further guidance on this point as needed.
+          </li>
+        </IndentedUnorderedList>
+      </li>
+      <li><strong>NOT</strong> attempt to contact <AoUTitle/> participants.
+      </li>
+      <li><strong>NOT</strong> make copies of or download individual-level data outside of
+        the <AoUTitle/> environment without approval from RAB.
+        <IndentedUnorderedList>
+          <li>I will not take screenshots or attempt any other way of
+            copying individual-level data.
+          </li>
+        </IndentedUnorderedList>
+      </li>
+
+      <li><strong>NOT</strong> redistribute or publish registered or controlled tier <AoUTitle/> data including
+        aggregate statistics that are more granular than buckets of 20 individuals without explicit approval from
+        the <AoUTitle/>.
+      </li>
+      <li><strong>NOT</strong> attempt to link participant-level All of Us data from the registered or controlled
+        tier with participant-level data from other sources without explicit permission from the All of Us Research
+        Program.</li>
+      <li><strong>NOT</strong> use <AoUTitle/> data for marketing purposes.
+        <IndentedUnorderedList>
+          <li>“Marketing” means a communication about a product or service that encourages recipients of the
+            communication to purchase or use the product or service (US 45 CFR 164.501).
+          </li>
+        </IndentedUnorderedList>
+      </li>
+      <li><strong>NOT</strong> represent that the <AoUTitle/> endorses or approves of my research unless such
+        endorsement is expressly provided.
+      </li>
+    </IndentedUnorderedList>
+
+    <SecondHeader>Data Disclaimer:</SecondHeader>
+    <p>The <AoUTitle/> does not guarantee the accuracy of the data in
+      the <AoUTitle/> database. The <AoUTitle/>
+      does not guarantee the performance of the software in the <AoUTitle/>
```

Comment:
This has a missing space, can be fixed by moving `the <AoUTitle/>` to the next line.